src/lib/utils/formatEventCTA.ts

```diff
@@ -0,0 +1,15 @@
+export function formatEventCTA(input: string): string {
```

Comment:
i'd prefer this file to be named `utils/events.ts` so any additional event utils make sense to live in here too